src/Models/clients.js

```diff
@@ -38,7 +45,7 @@ const clientSchema = new mongoose.Schema({
         maxlength: 15,
         trim: true 
       },
-      office: {
+      offices: {
```

Comment:
@KristomeisterDev Debe ser en singular ya que hace referencia a una sola office
```suggestion
      office: {
```

Reply:
ok, por eso me saco de onda cuando me dijiste que la pusiera en plural, pero a lo mejor nos confundimos de codigo, lo paso a singular.